VERSION

```diff
@@ -0,0 +1 @@
+0.11.0
```

Comment:
I think we can get rid of this now.

Reply:
Removed in [`5be38e1` (#210)](https://github.com/chainguard-dev/bincapz/pull/210/commits/5be38e1648fa83c7a9330a33a247efd1461a173e).